.idea/dbnavigator.xml

```diff
@@ -0,0 +1,454 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Probably didn't want to include this file right?

Reply:
Deleted!